File: hello.php
<!DOCTYPE html>
<html lang="ja">

    <head>
        <meta charset="utf-8">
        <title>hello</title>
    </head>

    <body>
        <?php
            // 関数定義の練習
            function FizzBuzz (int $sum_number): string{
                if ($sum_number%3 == 0 && $sum_number%5 == 0){
                    return "FizzBuzz";
                } elseif ($sum_number%3 == 0) {
                    return "Fizz";
                } elseif ($sum_number%5 == 0) {
                    return "Buzz";
                } else {
                    return (string) $sum_number;
                    /*
                    $sum_numberをそのまま返すとintなのでエラーになる
                    (返り値の型を宣言しているため)
                    */
                }
            }

            echo "Hello World！"."\n";

            // PHPの配列は連想配列
            $array_number = array();

            for ($i = 0; $i < 5; $i ++) {
                $array_number[$i] = rand(0,100);
            }
            
            $sum_number = array_sum($array_number);
            
            $res = FizzBuzz($sum_number);
            echo $res."\n";
        ?>
    </body>

</html>

File: hello.go
package main

import(
	"fmt"
	"math/rand"
	"time"
	"strconv"
)

// 関数定義の練習
func FizzBuzz(sum_number int) string{
	/*
	resを予め宣言して""で初期化した場合
	resは文字列型なので数値を代入できない
	strconvを使って明示的に変換する
	(pythonとかだと入っちゃう)
	*/
	res := ""
	if sum_number%3 == 0 && sum_number%5 == 0 {
		res = "FizzBuzz"
	} else if sum_number%3 == 0 {
		res = "Fizz"
	} else if sum_number%5 == 0 {
		res = "Buzz"
	} else {
		res = strconv.Itoa(sum_number)
	}
	return res
}

func main() {
	fmt.Println("Hello, World!")

	// 固定長の配列
	var array_number[5] int

	// 可変長の配列はスライス
	// var slice_number []int

	// 乱数を使うにはシードを設定
	rand.Seed(time.Now().UnixNano())

	// ループ処理
	for i:=0; i<5; i++{
		array_number[i] = rand.Intn(100)
	}

	// 配列を合計する関数がない
	sum_number := 0
	for _, x := range array_number{
		sum_number += x
	}

	// 宣言!しないと代入できない
	res := FizzBuzz(sum_number)

	fmt.Println(res)
}